idaes/generic_models/properties/core/examples/CO2_H2O_MEA_Ideal_VLE_Final.py

```diff
@@ -0,0 +1,195 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Property package for MEA Carbon Capture System - Stripper Reboiler
+
+CO2-H2O-MEA System
+Thermodynamic Model: Ideal liquid and vapor phase VLE with non-volatiles
+"""
+import sys
+import os
+import logging
+
+# Import units package from Pyomo
+from pyomo.environ import units as pyunits
+
+# Import IDAES cores
+from idaes.core import LiquidPhase, VaporPhase, Component
+from idaes.core.phases import PhaseType as PT
+from idaes.generic_models.properties.core.state_definitions import FTPx
+from idaes.generic_models.properties.core.eos.ideal import Ideal
+from idaes.generic_models.properties.core.phase_equil import smooth_VLE
+from idaes.generic_models.properties.core.phase_equil.bubble_dew import \
+        IdealBubbleDew
+from idaes.generic_models.properties.core.phase_equil.forms import fugacity
+import idaes.generic_models.properties.core.pure.Perrys as Perrys
+import idaes.generic_models.properties.core.pure.NIST as NIST
+
+# Import custom methods required to build the property package
+from idaes.generic_models.properties.core.examples import CustomMethods_Final,\
```

Comment:
1. I think these custom methods need more descriptive names.
2. I don't think these should be in separate files, and definitely shouldn't be importable directly from `examples`.